app.js

```diff
@@ -69,6 +93,24 @@ app.use(methodOverride('_method'))
 app.use(cookieParser('secret'))
 app.use(flash())
 
+//Set up HTTP Parameter Pollution
+app.use(require('hpp')())
+
+//Set up Helmet
+//app.use(helmet.hsts()) //HTTP Strict-Transport-Security enable once ssl
+app.use(helmet.frameguard())
+app.use(helmet.noSniff())
+app.use(helmet.ieNoOpen())
+app.use(helmet.hidePoweredBy({setTo: 'Whisky Powered.'}))
```

Comment:
We're opensource tho

Reply:
or is it private?

Reply:
It's true that we're open source, but anyone trying to utilize that header technique to gain information about our stack would be malicious. It provides some minimal security with no cons to user experience.